t/19-transaction.cpp

```diff
@@ -39,27 +40,29 @@ TEST_CASE("transaction", "[connection]") {
     auto port_str = boost::lexical_cast<std::string>(port);
     auto server = ts::make_server({"redis-server", "--port", port_str});
     ep::wait_port<ep::Kind::TCP>(port);
-    asio::io_service io_service;
+    asio::io_context io_service;
 
     r::command_container_t tx_commands = {
-        r::single_command_t("MULTI"), r::single_command_t("INCR", "foo"),
-        r::single_command_t("INCR", "bar"), r::single_command_t("EXEC"),
+        r::single_command_t("MULTI"),
+        r::single_command_t("INCR", "foo"),
+        r::single_command_t("INCR", "bar"),
+        r::single_command_t("EXEC"),
     };
     r::command_wrapper_t cmd(tx_commands);
 
     std::promise<result_t> completion_promise;
     std::future<result_t> completion_future = completion_promise.get_future();
 
     asio::ip::tcp::endpoint end_point(
-        asio::ip::address::from_string("127.0.0.1"), port);
+        asio::ip::make_address("127.0.0.1"), port);
```

Comment:
These 4 lines are repeated millions of times. I think they can be converted to a function and call anywhere need